tooling/did-method-registry-entry.yml

```diff
@@ -41,6 +41,18 @@ properties:
     description: https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry
     type: string
     maxLength: 512
+  implementation:
+    description: A URL to an implementation of the DID Method
+    type: string
+    maxLength: 512
+  conformanceReport:
+    description: A URL to a DID Test Suite Conformance Report for the DID Method
```

Comment:
I wonder if we might use `did:key` as an example instead of `did:example`... and provide real URLs for these new optional entries.

Adding optional fields, that are empty in the example might not be the strongest move here.

Reply:
Yep, I was thinking the same thing.